src/test/js/shared/idam_helper.js

```diff
@@ -214,8 +214,14 @@ class IdamHelper extends Helper {
             .catch(err => err);
     }
 
-    generateRandomText() {
-        return Math.random().toString(36).substr(2, 5);
+    generateRandomText(stringLength = 5) {
```

Comment:
I see what this is doing, but why was it needed?

Reply:
The initial method used to generate random alpha_numeric_ characters. The requirements in the ticket specify to disallow numbers in names and this method is used to generate random strings to be used as names.